solutions/gold/ac-smallmultiple.mdx

```diff
@@ -15,6 +15,39 @@ author: Chongtian Ma
 
 **Time Complexity:** $\mathcal{O}(K)$
 
+## Aproach
```

Comment:
put this above the implementation header and below the official analysis section

Reply:
Done